pkg/api/datasources.go

```diff
@@ -231,6 +238,27 @@ func AddDataSource(c *models.ReqContext, cmd models.AddDataSourceCommand) respon
 	})
 }
 
+func (hs *HTTPServer) StoreDataSourceInVCS(ctx context.Context, ds models.DataSource) error {
+	if gitops, ok := hs.Cfg.FeatureToggles["gitops"]; !ok || !gitops {
+		return nil
+	}
+
+	dsJson, err := json.Marshal(ds)
+	if err != nil {
+		return err
+	}
+
+	vobj := vcs.VersionedObject{
+		ID:        ds.Uid,
+		Version:   "",
+		Kind:      vcs.Datasource,
+		Data:      dsJson,
+		Timestamp: time.Now().Unix(),
```

Comment:
Do we really need to set the timestamp when storing? At least for the current implementation of the Github plugin, it's not used but replaced with the commit date.

Reply:
I guess we don't